src/api/repos/releases.rs

```diff
@@ -265,6 +265,28 @@ pub struct CreateReleaseBuilder<'octo, 'repos, 'handler, 'tag_name, 'target_comm
     draft: Option<bool>,
     #[serde(skip_serializing_if = "Option::is_none")]
     prerelease: Option<bool>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    make_latest: Option<MakeLatest>,
+}
+
+#[derive(Debug, Clone, Copy, serde::Serialize)]
+#[serde(rename_all = "lowercase")]
+pub enum MakeLatest {
+    True,
+    False,
+    Legacy,
+}
+
+impl std::fmt::Display for MakeLatest {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        let text = match self {
+            Self::False => "false",
```

Comment:
FYI: just in case you are not aware, but you can delegate this "manual" string mapping to serde, since you use already the `rename_all = "lowercase"` attribute, serde will take care of renaming the enum variants accordingly.

You can find an example [here](https://github.com/curlz-rs/curlz/blob/main/src/curlz/data/request/http_method.rs#L21) and [a test here](https://github.com/curlz-rs/curlz/blob/main/src/curlz/data/request/http_method.rs#L52)

Reply:
Thanks, I suspected there was a better way to do it :)

Reply:
Of course while that solves the problem for serde, it doesn't let you easily print the value in other contexts, I don't think you should have go through a whole serialisation process just to print the value, so I think having the display impl is good regardless of serde.